rootfs/etc/haproxy/template/haproxy.tmpl

```diff
@@ -108,6 +108,18 @@ frontend httpfront
 {{ range $server := $ing.HAServers }}
 {{ if $server.UseHTTP }}
 {{ range $location := $server.Locations }}
+    http-request set-var(req.ingress_orig_path) path
```

Comment:
my assumption is that you are using a var because the path is being changed before match on use_backend, is it right? Note that you are using the same var name to different hosts and paths.

Reply:
Right, I was finding that after the `reqrep` we could not execute the same `acl` against the path when choosing the backend. OH! Probably do need to move the variable setting to outside the loop though, good call there.